tests/integration/cli/tests/config.rs

```diff
@@ -1,16 +1,41 @@
-use std::path::Path;
+use std::{fs::OpenOptions, io::Write, path::Path};
 
 use assert_cmd::Command;
 use predicates::str::contains;
 use tempfile::TempDir;
 use wasmer_integration_tests_cli::get_wasmer_path;
-use wasmer_registry::WasmerConfig;
 
 fn setup_wasmer_dir() -> TempDir {
     let temp = TempDir::new().unwrap();
 
-    let config_path = WasmerConfig::get_file_location(temp.path());
-    WasmerConfig::default().save(&config_path).unwrap();
+    // The config path and the config contents themselves are manually crafted so that we don't
+    // depend on the cli crate.
+    //
+    // Eventually, this part of the config shall live on a freestanding crate - perhaps added to
+    // `wasmer-config`.
+    let config_path = temp.path().join("wasmer.toml");
+
+    let contents = r#"
+telemetry_enabled = false
```

Comment:
Why false by default. This should be true

Reply:
It's the default we have when we create the config, both [in the `registry` crate](https://github.com/wasmerio/wasmer/blob/main/lib/registry/src/config.rs#L11) and [in the `cli` crate](https://github.com/wasmerio/wasmer/blob/main/lib/cli/src/config/mod.rs#L33)(which is what we currently use). Should I change the default to `true` there?